#Boost checks Python include in $(prefix)/include/python$(version)
#some cases it is in $(prefix)/include/python$(version)m
#see more: https://svn.boost.org/trac/boost/ticket/11120
# Aug 27th 2016 by B. Hajgato (Free Univeristy Brussels - VUB)
--- boost_1_60_0/tools/build/src/tools/python.jam.org	2015-10-16 20:55:36.000000000 +0200
+++ boost_1_60_0/tools/build/src/tools/python.jam	2016-07-27 16:20:52.549560154 +0200
@@ -539,7 +539,16 @@
     }
     else
     {
-        includes ?= $(prefix)/include/python$(version) ;
+        if  not($(prefix)/include/python$(version)m)
+        {   
+           debug-message "Used include path: $(prefix)/include/python$(version)m" ;
+           includes ?= $(prefix)/include/python$(version)m ;
+        }
+        else
+        {
+           debug-message "Used include path: $(prefix)/include/python$(version)" ;
+           includes ?= $(prefix)/include/python$(version) ;
+        }
 
         local lib = $(exec-prefix)/lib ;
         libraries ?= $(lib)/python$(version)/config $(lib) ;
